### ISSUE TITLE
Use of Arabic within images

### ISSUE THREAD

**Issue by jwld** [FACN1/VolunteerApp#29]
The Arabic should be actual text wherever possible - any Arabic within an image is impossible to copy, run through Google translate, read with a screen reader etc.
